pkg/services/ngalert/remote/remote_primary_forked_alertmanager.go

```diff
@@ -37,13 +37,18 @@ func (fam *RemotePrimaryForkedAlertmanager) ApplyConfig(ctx context.Context, con
 	return nil
 }
 
-// TODO: save the new configuration hash in memory.
```

Comment:
We're not storing the configuration hash, the sync loop for configurations will not be here.